oozie-to-airflow/utils/xml_utils.py

```diff
@@ -71,3 +84,18 @@ def find_nodes_by_attribute(root, attr, val, tag=None) -> List[ET.Element]:
         if attr in node.attrib and node.attrib[attr] == val:
             matching_nodes.append(node)
     return matching_nodes
+
+
+def get_text_or_default(root: ET.Element, tag: str, params: Dict[str, str], default: str = None):
```

Comment:
I think the name of this method could be better. Currently it doesn't indicate that any EL replacing takes place inside.

Reply:
What do you think about `get_el_text_or_default`?

Reply:
Yeah, it's better.
I'm not sure about the `or_default` part. In general I dislike words like `and`, `or` in method names.

Maybe simply `get_tag_el_text`?

Reply:
I updated the PR.